colossalai/ppinference/__init__.py

```diff
@@ -0,0 +1,4 @@
+from .engine import PPInferEngine
```

Comment:
Can you rename this folder to `inference/pipeline`? As there may be other inference modules like `inference/tensor_parallel`, `inference/quant`.

Reply:
done